modules/monitoring/manifests/checks/sidekiq.pp

```diff
@@ -45,6 +45,21 @@
     args      => '--dropfirst -60',
   }
 
+  @@icinga::check::graphite { 'check_publishing_api_downstream_high_queue_latency':
```

Comment:
I'll approve this, but I think it would be better as a healthcheck.

Using [`SidekiqQueueLatencyCheck`](https://github.com/alphagov/govuk_app_config/blob/master/docs/healthchecks.md#sidekiqqueuelatencycheck) it's in the app itself rather than in Puppet config, you get more customisation and a clearer message in Icinga which includes the actual latency and the thresholds.

Reply:
Thanks for taking a look Tom. I guess using the healthcheck wouldn't immediately get to the point where Pagerduty can be used for notifications (unless I'm missing something), but maybe that's something that can be sorted as well...

I'll take a look at adding the healthcheck, and close this if it works out...

Reply:
Hmm true, you would have to assign an alert to the critical health check which might call on other things.

Reply:
The healthcheck is deployed now, so I'll have a look at notifications in the context of healthchecks... at some point.